test/facts/get-pondless-area.test.js

```diff
@@ -0,0 +1,33 @@
+const getPondlessAreaFact = require('../../facts/get-pondless-area')
+
+describe('get-pondless-area fact', () => {
+  test('should return total area if there are no pond features', async () => {
+    const fakeAlamanc = {
+      factValue: () => {
+        return { totalArea: 50, areaFeatures: [] }
+      }
+    }
+    const result = await getPondlessAreaFact({}, fakeAlamanc)
+    expect(result).toBe(50)
+  })
+
+  test('should return total area if there are features which are not ponds', async () => {
+    const fakeAlamanc = {
+      factValue: () => {
+        return { totalArea: 50, areaFeatures: [{ type: 'not a pond', areaCovered: 20 }] }
+      }
+    }
+    const result = await getPondlessAreaFact({}, fakeAlamanc)
+    expect(result).toBe(50)
+  })
+
+  test('should return adjusted area if there are features which are ponds', async () => {
+    const fakeAlamanc = {
+      factValue: () => {
+        return { totalArea: 50, areaFeatures: [{ type: 'pond', areaCovered: 20 }] }
```

Comment:
Again, I would test with at least two items in the `areaFeatures` array to confirm that they are combined correctly.